app/controllers/api/v1/customers_controller.rb

```diff
@@ -6,7 +6,7 @@ module Api
   module V1
     class CustomersController < Api::V1::BaseController
       include AddressTransformation
-      include Api::V1::QueryParam
+      include ReusableQueryParams
```

Comment:
What about `ExtraFields`?

Reply:
Sure

Reply:
Sure, I guess I thought it could become a small library. But I'm starting to understand your logic of "its not a library yet"